sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs

```diff
@@ -38,10 +38,17 @@ public RecognizeContentOptions()
         public string Language { get; set; }
 
         /// <summary>
+        /// <para>
         /// Custom page numbers for multi-page documents(PDF/TIFF). Input the number of the
         /// pages you want to get OCR result. For a range of pages, use a hyphen.
         /// Separate each page or range with a comma.
+        /// For sample "1-3", {"1", "3", "5-7"}.
+        /// </para>
+        /// <para>
+        /// Although this collection cannot be set, it can be modified.
```

Comment:
Not sure I understand, we must have a `setPages()` method correct to assign `pages` a value?

Reply:
.NET doesn't have `setXX` methods. what this is saying is you can't initialize or overwrite, i.e. `Pages = new List<string>() {}`
but you can add things to the `Pages` property. You could do `Pages.Add("1-2")` or `Pages = { "1-2" }`.

This is because in .NET our collections are not settable (guideline). I am providing more information in case the user needs it.
I copied this from KV